app/models/date_formatter.rb

```diff
@@ -0,0 +1,7 @@
+class DateFormatter
+  def self.format(date)
+    dtime = date.to_date.asctime.gsub("00:00:00 ", "")
+    dtime[8] = '0' if dtime[8] == ' '
+    dtime
```

Comment:
Nice. If you wanted to simplify this, the `strftime` method is really powerful and could more precisely spit out what you want without having to use gsub: https://ruby-doc.org/core-2.6.2/Time.html#method-i-strftime